homeassistant/components/onvif/camera.py

```diff
@@ -341,6 +362,16 @@ def setup_ptz(self):
         else:
             _LOGGER.debug("Camera '%s' doesn't support PTZ.", self._name)
 
+    async def async_perform_reboot(self):
+        """Perform a SystemReboot action on the camera."""
+        try:
+            _LOGGER.debug("Calling SystemReboot")
+            ret = await self._camera.devicemgmt.SystemReboot()
+            _LOGGER.debug("Camera '%s' Reboot command returned '%s'", self._name, ret)
+        except exceptions.ONVIFError as err:
+            if "Bad Request" in err.reason:
```

Comment:
Shouldn't we log anything in other error cases?

Reply:
true, fixed